posthog/api/plugin.py

```diff
@@ -604,6 +606,12 @@ def get_delivery_rate_24h(self, plugin_config: PluginConfig):
         else:
             return None
 
+    def get_error(self, plugin_config: PluginConfig) -> None:
+        # Reporting the single latest error is no longer supported: use app
+        # metrics (for fatal errors) or plugin log entries (for all errors) for
+        # error details instead.
+        return None
```

Comment:
This just turns [this block](https://github.com/PostHog/posthog/blob/f476f67f5cc5a10368712e94aba5807407f2d0cb/frontend/src/scenes/plugins/tabs/apps/AppView.tsx#L127-L146) effectively into a constant with the non-error state which is probably good enough for now but also not ideal.

If we wanted to preserve the existing behavior, we could check the log entries table in a similar fashion to how delivery rate is added to this serializer. We wouldn't have the same degree of detail here from log entry data (though we weren't actually using the data we were writing other than as a truthy value), so the type of this would need to change on the frontend to accommodate that.

I also didn't start to remove [the `error` field on `PluginConfigType`](https://github.com/PostHog/posthog/blob/0cafec3eb4a99e4dc2585121d70af95d27c2e9d1/frontend/src/types.ts#L1528) and all the related bits (`PluginErrorType`, etc.) since I don't really know my way around the frontend yet. Happy to do that as a follow up to learn more.